underpants.js

```diff
@@ -220,7 +350,21 @@ var _ = {};
 * Examples:
 *   _.pluck([{a: "one"}, {a: "two"}], "a") -> ["one", "two"]
 */
-
+_.pluck = function(arr, prop) {
+    // Iterates over 'arr'
+    var output = _.map(arr, function (element) {
+        // Iterates over objects in 'arr'
+        for (var [key, value] of Object.entries(element)) {
+            // Finds 'prop' and returns its value
+            var val;
+            key === prop ? val = value : null;
+            return val;
+        }
+    });
+
+    // Returns a new array
+    return output;
```

Comment:
```suggestion
    return _.map(arr, function (element) {
        // Iterates over objects in 'arr'
        for (var [key, value] of Object.entries(element)) {
            // Finds 'prop' and returns its value
            var val;
            key === prop ? val = value : null;
            return val;
        }
    });
```

It is possible, and is not a bad coding practice at all, to just return a map like this directly.